.changeset/metal-swans-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/components": minor
+---
+
+Deprecate utility components.
```

Comment:
It might be helpful for consumers if we listed all the deprecated components explicitly here.

Reply:
In addition to a list of deprecated components, let's also include instructions on how to run the codemod here.